File: results/py/REMOVE_MINIMUM_NUMBER_ELEMENTS_NO_COMMON_ELEMENT_EXIST_ARRAY.cpp
int f_filled ( int a [ ] , int b [ ] , int n , int m ) {
  map < int , int > countA ;
  map < int , int > countB ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) countA [ a [ i ] ] = countA . find ( a [ i ] ) + 1 ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) countB [ b [ i ] ] = countB . find ( b [ i ] ) + 1 ;
  int res = 0 ;
  for ( int x : countA ) {
    if ( countB . count ( x ) > 0 ) res += min ( countA [ x ] , countB [ x ] ) ;
  }
  return res ;
}

File: results/java_codex/REMOVE_MINIMUM_NUMBER_ELEMENTS_NO_COMMON_ELEMENT_EXIST_ARRAY.java
int minRemove(int a[], int b[], int n, int m) {
        unordered_map<int, int> countA;
        unordered_map<int, int> countB;
        for (int i = 0; i < n; i++) {
            if (countA.find(a[i]) != countA.end()) countA[a[i]]++;
            else countA[a[i]] = 1;
        }
        for (int i = 0; i < m; i++) {
            if (countB.find(b[i]) != countB.end()) countB[b[i]]++;
            else countB[b[i]] = 1;
        }
        int res = 0;
        for (auto x : countA) {
            if (countB.find(x.first) != countB.end()) res += min(countB[x.first], x.second);
        }
        return res;
    }